resources/assets/tconstruct/lang/fr_FR.lang

```diff
@@ -0,0 +1,748 @@
+################
+#    Blocs     #
+################
+
+tile.tconstruct.tooltables.craftingstation.name=Station de réalisant
```

Comment:
Hello, I'm french, here is what I would write :
craftingstation = Table d'artisanat